### PR TITLE
Difficulty

### LINKED ISSUE #10: Sélection de la difficulté

### LINKED ISSUE #26: Navigation entre menus

### PR DESCRIPTION
**Pull request by Mana-One**
closes #26, closes #10

### CODE REVIEW

**Comment by Mana-One** [comment]
added nav between main title and difficulty screen
added option to go back to a previous screen
updated Keyboard to handle integer and ENTER_KEY inputs

**Comment by adem-m** [comment]
Seems great but I merge my PR first so I don't have to deal with conflicts, cheh

**Comment by adem-m** [comment]
Oh, you don't have some lucky boy. Let's merge it

File: src/main/java/com/esgi/DifficultyGameMode.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    }
+
+    @Override
+    void init() {
+        System.out.println( "Veuillez choisir un niveau de difficulté:\n" );
+
+        int index = 1;
+        for( Difficulty difficulty: Difficulty.values() ){
+            System.out.println( String.format( "%d - %s", index, difficulty ));
+            index += 1;
```
Comment (adem-m) [inline src/main/java/com/esgi/DifficultyGameMode.java:29]: Can be changed to `System.out.println( String.format( "%d - %s", index++, difficulty ));`
Index may have to start from 0

**Review by adem-m — CHANGES_REQUESTED** [review]

**Pull request merged**